content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md

```diff
@@ -562,6 +562,62 @@ Flagsmith does not currently support this integration. Create a ticket with Flag
 {{% /tab %}}
 {{< /tabs >}}
 
+
+### Kameleoon integration
+
+{{< tabs >}}
+{{% tab "Browser" %}}
+
+After creating and initializing Kameleoon SDK, subscribe to `Evaluation` event using the `onEvent` handler.
+
+For more information about SDK, see [Kameleoon JavaScript SDK documentation][1].
+
+```javascript
+client.onEvent(EventType.Evaluation, ({ featureKey, variation }) => {
+  datadogRum.addFeatureFlagEvaluation(featureKey, variation.key);
+});
+```
+
+
+[1]: https://developers.kameleoon.com/feature-management-and-experimentation/web-sdks/js-sdk
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+Kameleoon does not support this integration. Contact ~kameleoon support~ to request this feature
```

Comment:
**💬 suggestion:** ‏this renders "kameleoon support" as stroked text: ~kameleoon support~. Maybe change it to italic, and/or maybe at a link to Kameleoon support?

```suggestion
Kameleoon does not support this integration. Contact _Kameleoon support_ to request this feature
```